### ISSUE TITLE
Unnecessarily exported identifiers inspection

### ISSUE THREAD

**Issue by zolotov** [go-lang-plugin-org/go-lang-idea-plugin#1843]
The same funtionality as implemented in https://github.com/isaiah/unexport

**Comment by dlsniper** [comment]
I think this inspection could work only if the project inspected is an application not a library. And maybe it should be turned off by default or a `WEAK_WARNING`.

**Comment by zolotov** [comment]
@dlsniper indeed, it could be weak_warning by default or enabled for not_on_the_fly_mode only
